erpnext/stock/doctype/stock_ledger_entry/stock_ledger_entry.py

```diff
@@ -16,6 +16,13 @@ class StockFreezeError(frappe.ValidationError): pass
 exclude_from_linked_with = True
 
 class StockLedgerEntry(Document):
+	def autoname(self):
+		"""
+		Temporarily name doc for fast insertion
+		name will be changed using autoname options (in a scheduled job)
+		"""
+		self.name = frappe.generate_hash(txt="", length=10)
```

Comment:
Keep auto name field as blank in the stock ledger entry doctype

Reply:
Autoname can't be blank since its primary key (frappe will default it hash as primary)